packages/framework-provider-local/src/library/read-model-adapter.ts

```diff
@@ -24,10 +25,10 @@ export async function fetchReadModel(
   readModelName: string,
   readModelID: UUID
 ): Promise<ReadModelInterface> {
-  const response = await db.query({ typeName: readModelName, value: { id: readModelID } })
+  const response = await db.query({ typeName: readModelName, 'value.id': readModelID })
```

Comment:
This is the way of properly query nested properties like value.id

Reply:
Hey @davidverdu is this on the nedb docs?  Can we add it as a comment?

Reply:
Hi @juanjoman here is explained in [nedb documentation](https://github.com/louischatriot/nedb#finding-documents) . The dot notation 'value.id': readModelID is used to match properties with values (what we want to do on these queries), whereas the nested notation value: {id: readModelID} does a deep-comparison. I understand that if the model has more properties in value and you don't put them in the query, the query will have an unexpected behavior. It could return the value or not. By using the dot notation we ensure that the results are returned, regardless of the amount of properties present in the model